modules/nextflow/src/main/groovy/nextflow/k8s/K8sDriverLauncher.groovy

```diff
@@ -117,7 +122,8 @@ class K8sDriverLauncher {
         this.interactive = name == 'login'
         if( background && interactive )
             throw new AbortOperationException("Option -bg conflicts with interactive mode")
-        this.config = makeConfig(pipelineName)
+        this.configObject = makeConfig(pipelineName)
+        this.config = configObject.toMap()
         this.k8sConfig = makeK8sConfig(config)
```

Comment:
Bit ugly to keep both `config` and `configObject`. Think this point makes sense to get rid of the first. 

The main difference with the when navigating a `ConfigObject` it automatically creates the request attribute, so for example if you do `configObject.foo`  foo becomes a nested empty object. It should be used `configObject.isSet('foo')` instead